### PR TITLE
Further UR5 refactoring

### PR DESCRIPTION
**Pull request by paulbovbel**
- Add stow script to husky_control
- Move ros_control arm configuration loading to husky_gazebo, since it's only used in simulation

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -7,6 +7,8 @@ catkin_package()
 
 roslaunch_add_file_check(launch)
 
-install(DIRECTORY config launch
+catkin_install_python(PROGRAMS scripts/stow_ur5.py DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
```
Comment (mikepurvis) [inline CMakeLists.txt:10]: Best to leave off the `.py` extension; gives the freedom to change the impl later, and a nicer command-line interface, eg:

```
rosrun husky_control stow_ur5
```

File: scripts/stow_ur5.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
```
Comment (mikepurvis) [inline scripts/stow_ur5.py:1]: License header, please.

File: scripts/stow_ur5.py [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+import time
+import roslib; roslib.load_manifest('ur_driver')
```
Comment (mikepurvis) [inline scripts/stow_ur5.py:3]: Remove

File: scripts/stow_ur5.py (hunk 2 above)
Comment (paulbovbel) [inline scripts/stow_ur5.py:1]: Didn't see your notes before I merged. Who wrote this originally?

File: scripts/stow_ur5.py (hunk 3 above)
Comment (paulbovbel) [inline scripts/stow_ur5.py:3]: @TheDash , please fix.

File: CMakeLists.txt (hunk 1 above)
Comment (paulbovbel) [inline CMakeLists.txt:10]: +1

File: scripts/stow_ur5.py (hunk 3 above)
Comment (UltronDestroyer) [inline scripts/stow_ur5.py:3]: What's the issue here, it needs license header?

Comment (paulbovbel) [inline scripts/stow_ur5.py:3]: http://answers.ros.org/question/115346/what-does-roslibload_manifest-do/?answer=115681#post-id-115681

Comment (UltronDestroyer) [inline scripts/stow_ur5.py:3]: Ah, ok.

**Pull request merged**